Source/WebCore/rendering/Grid.h

```diff
@@ -89,6 +89,9 @@ class Grid final {
     void setupGridForMasonryLayout();
     unsigned maxRows() const { return m_maxRows; }
     unsigned maxColumns() const { return m_maxColumns; }
+
+    Vector<RenderBox*> autoMajorAxisAutoGridItems;
```

Comment:
Why this public member variable without an `m_` prefix? Also we don't use raw pointers any more.

Reply:
Going to remove this and re-perform the computation in `resolveIntrinsicTrackSizes`.